.travis.yml

```diff
@@ -26,7 +26,6 @@ matrix:
     - env: MODE=semantic PR_ONLY=false
```

Comment:
@veronicagg - should line number 16 be moved up to line 12? In that way we know that all of the required ones are together?

Reply:
sure, I can move it up.